runtime/pkg/dag/dag.go

```diff
@@ -39,7 +39,7 @@ func (d *DAG) Add(name string, dependants []string) *Node {
 		}
 	}
 
-	for newParent, _ := range dependantMap {
+	for newParent := range dependantMap {
```

Comment:
Disable this rule – previously, it was obvious that `newParent` was a key. Now, I'm in doubt whether it's a key or a value.